test/upgrade/main.go

```diff
@@ -354,14 +354,18 @@ func checkHelmStatus(agonesVersion string) string {
 		log.Fatal("Could not Unmarshal", err)
 	}
 
-	// Remove the commit sha from the DevVersion i.e. from 1.46.0-dev-7168dd3 to 1.46.0-dev
+	// Remove the commit sha from the DevVersion i.e. from 1.46.0-dev-7168dd3 to 1.46.0-dev or 1.46.0
+	// for the case of this test running during a new Agones version release PR.
+	agonesRelease := ""
```

Comment:
(non blocking) in the future, may want to use a library like https://github.com/blang/semver (which we use in a few util scripts) to work with semantic versions, rather than regex things.